docs/format/source/format_release_notes.rst

```diff
@@ -1,23 +1,26 @@
 Release Notes
 =============
 
-2.3.0 (Upcoming)
+2.3.0 (April 23, 2021)
 ----------------
 
 - Add optional ``strain`` field to ``Subject``.
 - Add to ``DecompositionSeries`` an optional ``DynamicTableRegion`` called ``source_channels``.
 - Add to ``ImageSeries`` an optional link to ``Device``.
 - Clarify documentation for electrode impedance and filtering.
-- Add optional "continuity" field to `TimeSeries`.
-- Update hdmf-common-schema to version 1.2.0. Release notes:
-  - Add software process documentation.
-  - Fix missing dtype for ``VectorIndex``.
-  - Add new ``VocabData`` data type.
-  - Move ``Data``, ``Index``, and ``Container`` to base.yaml. This change does not functionally change the schema.
-  - ``VectorIndex`` now extends ``VectorData`` instead of ``Index``. This change allows ``VectorIndex`` to index other
-    ``VectorIndex`` types.
-  - The ``Index`` data type is now unused and has been removed.
-  - Fix documentation for ragged arrays.
+- Add optional "continuity" field to ``TimeSeries``.
+- Update hdmf-common-schema from 1.1.3 to version 1.5.0.
+  - The HDMF-experimental namespace was added, which includes the ``ExternalResources`` and ``EnumData``
+    data types. Schema in the HDMF-experimental namespace are experimental and subject to breaking changes at any time.
+  - Added experimental data type ``ExternalResources`` for storing ontology information / external resource references.
+  - Added experimental data type ``EnumData`` to store data from a set of fixed values.
+  - Changed dtype for datasets within ``CSRMatrix`` from 'int' to 'uint' and added missing `data_type_inc: Container`
+    to the ``CSRMatrix`` type.
+    Added data type ``SimpleMultiContainer``, a Container for storing other Container and Data objects together.
+  - Fixed missing dtype for ``VectorIndex``.
```

Comment:
```suggestion
  - Added data type ``AlignedDynamicTable``, a DynamicTable type with support for categories (or sub-headings) each described by a separate DynamicTable.
  - Fixed missing dtype for ``VectorIndex``.
```

Reply:
Oops I forgot this as I was making readthedocs for HDMF common 1.5.0 work. Thanks!